content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -521,6 +521,14 @@ The {{< glossary_tooltip text="control plane" term_id="control-plane" >}} adds t
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Example: EndpointsLastChangeTriggerTime = "endpoints.kubernetes.io/last-change-trigger-time"
```

Comment:
This isn't a valid example. Please fix it: make sure that the key is correct, and that the example value is a valid timestamp.

Reply:
@sftim Made changes